docs/source/conf.py

```diff
@@ -79,6 +81,7 @@
     # these are not defined in https://docs.python.org/3/objects.inv
     ("py:class", "socket.AddressFamily"),
     ("py:class", "socket.SocketKind"),
+    ("py:class", "Buffer"),  # collections.abc.Buffer, in 3.12
```

Comment:
huh, I'm a bit surprised it's not available in the inv file given that it's in the online docs https://docs.python.org/3.12/library/collections.abc.html#collections.abc.Buffer. But I guess it's just a matter of waiting for the release

Reply:
It's in the `inv` file, but we're linking to `/3/` which is still 3.11.